calibration/ADCPAM/CGINS-ADCPAM-50865__20130802.csv

```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+50865,CC_scale_factor1,0.61,DVL Bay 0548; FCP 3304-00150-00112.docx
+50865,CC_scale_factor2,0.61,
+50865,CC_scale_factor3,0.61,
+50865,CC_scale_factor4,0.61,
```

Comment:
Looks good (SNW) – I added DVL Bay S/N and FCP file.

Reply:
Would this earlier one still refer to same FCP?  (RGT)